pkg/config/config.go

```diff
@@ -3,22 +3,47 @@ package config
 import "github.com/urfave/cli/v2"
 
 var (
+	// Github - github configuration
 	Github struct {
-		Token         string
-		Refresh       int64
-		Repositories  cli.StringSlice
-		Organizations cli.StringSlice
-		ApiUrl        string
+		AppID             int64  `split_words:"true"`
```

Comment:
you can use cli.Int64Slice, the cli lib split automatly the string to int64 slice. not mandatory